classes/options-general.php

```diff
@@ -844,7 +844,7 @@ class="radio-thumbnail<?php if ( 'TERRAIN' == $map_type ) {
 																?>
 															</select><br/>
 															<small>
-																<em><?php printf( __( 'To add your own theme, upload your own CSS file to a new directory in your plugins folder called %s simplemap-styles%s.  To give it a name, use the following header in the top of your stylesheet:', 'simplemap' ), '</em><code>', '</code><em>' ); ?></em>
+																<em><?php printf( __( 'To add your own theme, upload your own CSS file to a new directory in your plugins folder called %s simplemap-styles%s.  To give it a name, use the following header in the top of your stylesheet:', 'SimpleMap' ), '</em><code>', '</code><em>' ); ?></em>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 339 characters

<!-- Comment by [SideCI](https://sideci.com) -->